src/services/Friend/FetchGroupAdmin.js

```diff
@@ -0,0 +1,91 @@
+import { Alert } from "react-native";
+import { firebase } from "../Firebase/Config";
+
+export function fetchGroupAdminIDs({ groupID, onSuccess, app = firebase }) {
+  return app
+    .firestore()
+    .collection("groups")
+    .doc(groupID)
+    .collection("admins")
+    .onSnapshot(
+      (querySnapshot) => {
+        const adminIDs = [];
+        const adminDatas = [];
+
+        querySnapshot.forEach((documentSnapshot) => {
+          adminIDs.push(documentSnapshot.id);
+        });
+
+        /*await app
+          .firestore()
+          .collection("users")
```

Comment:
You can delete this comment if you don't plan to use it since it is taking a lot of space